help/destinations/catalog/email-marketing/salesforce-pardot.md

```diff
@@ -0,0 +1,203 @@
+
+This [!DNL Adobe Experience Platform] [destination](/help/destinations/home.md) leverages the [[!DNL Account Engagement]](https://developer.salesforce.com/docs/marketing/pardot/guide/overview.html) API, to work with leads for your sales needs after activating them within a new [!DNL Salesforce Pardot] segment.
+
+[!DNL Salesforce Pardot] uses OAuth 2 with Client Credentials as the authentication mechanism to communicate with the [!DNL Account Engagement] API. Instructions to authenticate to your [!DNL Salesforce Pardot] instance are further below, in the [Authenticate to destination](#authenticate) section.
+
+## Use cases {#use-cases}
+
+To help you better understand how and when you should use the [!DNL Salesforce Pardot] destination, here is a sample use case that Adobe Experience Platform customers can solve by using this destination.
+
+### Send emails to contacts for marketing campaigns {#use-case-send-emails}
+
+The marketing department of an online platform wants to broadcast an email based marketing campaign to a curated audience of B2B leads. The platform's marketing team can add new leads / update existing lead information through Adobe Experience Platform, build segments from their own offline data, and send these segments to [!DNL Salesforce Pardot], which can then be used to send the marketing campaign email.
+
+## Prerequisites {#prerequisites}
+
+Refer to the sections below for any prerequisites that have to be setup / information that is to be gathered before working with the [!DNL Salesforce Pardot] destination.
+
+### Prerequisites in Experience Platform {#prerequisites-in-experience-platform}
+
+Before activating data to the [!DNL Salesforce Pardot] destination, you must have a [schema](/help/xdm/schema/composition.md), a [dataset](https://experienceleague.adobe.com/docs/platform-learn/tutorials/data-ingestion/create-datasets-and-ingest-data.html?lang=en), and [segments](https://experienceleague.adobe.com/docs/platform-learn/tutorials/segments/create-segments.html?lang=en) created in [!DNL Experience Platform].
+
+>[!TIP]
+>
+>The [!DNL Salesforce Pardot] destination uses the selected segment names, to create custom prospect fields in [!DNL Salesforce Pardot], and update them with the Experience Platform segment status. The destination automatically replaces whitespace or special characters with the underscore `(_)` character as a separator to comply with [!DNL Salesforce Pardot] requirements.
+>However, as a recommendation, when creating a segment, do not include whitespace or special characters in the segment name. Instead, use the underscore `(_)` character as a separator. For example instead of naming your segment as `Salesforce Pardot Segment 1` use `Salesforce_Pardot_Segment_1`.
```

Comment:
As discussed, I think that you can remove this suggestion, as we would expect perhaps customers to create segments that they reuse across multiple destinations.